app/models/case_log.rb

```diff
@@ -359,13 +367,12 @@ def ethnic_refused?
   end
 
   def receives_housing_related_benefits?
-    receives_housing_benefit_only? || receives_uc_with_housing_element_excl_housing_benefit? ||
-      receives_housing_benefit_and_universal_credit?
-  end
-
-  def benefits_unknown?
-    # 3: Don’t know
-    hb == 3
+    if collection_start_year <= 2021
+      receives_housing_benefit_only? || receives_uc_with_housing_element_excl_housing_benefit? ||
```

Comment:
Same question here

Reply:
This has no effect since all we do is remove a check on a removed value (for documentation purposes more than anything).